cache-config/testing/docker/trafficserver/Dockerfile

```diff
@@ -63,10 +63,12 @@ RUN if [[ ${OS_VERSION%%.*} -ge 8 ]]; then \
 			curl \
 			gcc-toolset-9 \
 			gcc-toolset-9-runtime \
+			geoip \
```

Comment:
Why add the `geoip` dependency to t3c integration test's trafficserver-building Dockerfile but not the CDN-in-a-Box cache Dockerfile?

Reply:
I see, the CiaB Cache file just capitalizes the dependency `GeoIP`